src/strawman/Link/Link.js

```diff
@@ -0,0 +1,19 @@
+var
+	kind = require('enyo/kind'),
+	Anchor = require('enyo/Anchor');
+
+module.exports = kind({
+	name: 'Link',
+	kind: Anchor,
+	classes: 'link',
+	spotlight: true,
+	handlers: {
```

Comment:
This might be overkill for strawman, but what do you think about throwing the tap handling code into a mixin, which we can share with both `Link` and also the `moonstone/Item` used in the moonstone samples list.